packages/core/content-releases/admin/src/pages/tests/ReleaseDetailsPage.test.tsx

```diff
@@ -1,49 +1,56 @@
-import { render, screen, server } from '@tests/utils';
+import { render, server, screen } from '@tests/utils';
 import { rest } from 'msw';
 
 import { ReleaseDetailsPage } from '../ReleaseDetailsPage';
 
-jest.mock('react-router-dom', () => ({
-  ...jest.requireActual('react-router-dom'),
-  useParams: jest.fn().mockImplementation(() => ({ id: '1' })),
-}));
+import { mockReleaseDetailsPageData } from './mockReleaseDetailsPageData';
 
 jest.mock('@strapi/helper-plugin', () => ({
   ...jest.requireActual('@strapi/helper-plugin'),
   // eslint-disable-next-line
   CheckPermissions: ({ children }: { children: JSX.Element}) => <div>{children}</div>
 }));
 
-describe('Release details page', () => {
-  it('renders correctly the heading content', async () => {
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn().mockImplementation(() => ({ releaseId: '1' })),
```

Comment:
please don't mock this, you can pass `initialEntries` to your render function iirc to provide a correct URL.